### PR TITLE
Remote test warning from nf-core lint

### PR DESCRIPTION
**Pull request by maxulysse**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: modules.json [hunk 1]
```diff
@@ -10,14 +10,14 @@
                         "git_sha": "5e34754d42cd2d5d248ca8673c0a53cdf5624905",
                         "installed_by": ["modules"]
                     },
-                    "bcftools/mpileup": {
+                    "bcftools/concat": {
```
Comment (JoseEspinosa) [inline modules.json:13]: Just changed the order, right?

**Review by JoseEspinosa — APPROVED** [review]
LGTM, just a small question :rocket:

File: modules.json (hunk 1 above)
Comment (maxulysse) [reply modules.json:13]: yes, it was all done by nf-core/tools, I was super worried, but @mashehu saved the day by explaining it was just a re-order

File: assets/methods_description_template.yml [hunk 2]
```diff
@@ -3,8 +3,6 @@ description: "Suggested text and references to use when describing pipeline usag
 section_name: "nf-core/sarek Methods Description"
 section_href: "https://github.com/nf-core/sarek"
 plot_type: "html"
-## TODO nf-core: Update the HTML below to your prefered methods description, e.g. add publication citation for this pipeline
```
Comment (FriederikeHanssen) [inline assets/methods_description_template.yml:6]: shouldn't we add some description on sarek here?

Comment (maxulysse) [reply assets/methods_description_template.yml:6]: I looked at the other pipelines, and nothing is there, the only difference I saw in some pipelines was the comments removed.
I'm not sure why it's in the TEMPLATE

**Pull request merged**